.github/workflows/qa.yml

```
@@ -0,0 +1,20 @@
+---
+
+name: Quality Assurance
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+
+  xmlllint:
+    runs-on: ubuntu-latest
+    steps:
+      - run: sudo apt install libxml2-utils
+      - uses: actions/checkout@v5
+      - run: find files -type f -name '*.xml' -exec xmllint --noout --pedantic {} \;
```

could it come in handy to have the detected issues grouped by type of text representation, i.e. e.g. ca:

```suggestion
    runs-on: ubuntu-latest
    strategy:
      matrix:
        suffix:
          - ''
          - _hiero
          - _st
          - _wt
    steps:
      - run: sudo apt install libxml2-utils
      - uses: actions/checkout@v5
      - run: find files -type f -name "*${suffix}.xml" -exec xmllint --noout --pedantic {} \;
        env:
          suffix: ${{ matrix.suffix }}
```

---

i don't see how `find` could be employed properly as it doesn't pass through failed exit codes.